### PR TITLE
Removes punctuation from search results before they are sent to CKAN

### PR DESCRIPTION
**Pull request by schalkventer**
image

image

### CODE REVIEW

**Comment by schalkventer** [comment]
@longhotsummer CKAN currently returns an error if the search string has punctuation in it. (Picked up by @guushoekman )

File: _includes/components/other-pages/SearchResult/partials/removePunctuation.js [hunk 1]
```diff
@@ -0,0 +1,3 @@
+export default function removePunctuation(string) {
+  return string.replace(/[^\w|^\s]/g, ' ');
```
Comment (longhotsummer) [inline _includes/components/other-pages/SearchResult/partials/removePunctuation.js]: just us `/[^\w\s]/`

**Comment by longhotsummer** [comment]
Perhaps just one definition of the utility function.

**Comment by schalkventer** [comment]
@longhotsummer Changes made.

**Comment by longhotsummer** [comment]
Great.

**Pull request merged**